src/samples/UWP/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.6.0-prerelease.210113001</Version>
+      <Version>2.8.5</Version>
```

Comment:
This version isn't the same as the others, is it intended?

Reply:
good catch @pictos, it should be 2.7.1 for all csproj @Arieldelossantos

Reply:
It was intended like that @pictos 
(See details here : https://github.com/unoplatform/Uno.Themes/issues/1142#issue-1843908717)

Reply:
yes, it's intended to use 2.8.5 on Sample UWP